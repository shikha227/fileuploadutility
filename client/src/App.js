import React, { Component } from "react";
import Upload from './componenet/Report/file upload.js';

import Navbar from "./componenet/layout/Navbar.js";
import Table from"./componenet/table/price.js";
 class App extends Component {
  render() {
    return (
              <div className="App">
        
       <Navbar/>
       <Upload/>
       
       </div>
    );
  }
}
export default App;