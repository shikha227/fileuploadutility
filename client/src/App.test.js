import { render, screen } from '@testing-library/react';
import ReactDOM from 'react-dom';
import App from './App';
import React from 'react';
import { shallow } from 'enzyme';
import Upload from './componenet/Report/file upload.js';
describe('Upload', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<Upload debug />);
  
    expect(component).toMatchSnapshot();
  });
it('button click should hide component', () => {
    const component = shallow(<Upload  />);
      const instance = component.instance();
    component
      .find('button')
      .simulate('click');

          expect(instance.state.error).toEqual('Please select file');

  });


});
