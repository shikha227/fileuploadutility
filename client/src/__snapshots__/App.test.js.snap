// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Upload should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
